### DISCUSSION TITLE
Foox

### DISCUSSION THREAD

**Discussion by git900000** [nikoniko78/scripts-java-physion#32] [Q&A] [answered]
Does anyone here know Foox?

**Comment by nikoniko78** [comment, marked as answer]
No, but I used to have contact. Pretty sure they left.

**Reply by git900000** [reply]
D'aww shoot, he used to make great things. Rest in peace, Foox, you will be missed. :(

**Reply by git900000** [reply]
I'm not saying that he might be dead! I'm just paying a tribute.

**Reply by slowpoke111** [reply]
He hasn't been active on the discord recently. When asked a few months ago he mentioned that he now does CAD and 3d printing.

**Reply by git900000** [reply]
Whaaaaa!? Cool! Did you ask him or did someone else?
